_pages/404.md

```diff
@@ -5,15 +5,12 @@ sitemap: false
 sidenav: false
 ---
 
-<div id="redirect-pending">
-We're checking <a target="blank" href="https://archive.ada.gov">Archive.ADA.gov</a> for you, please wait...
-</div>
-
-<div id="redirect-failed" style="display: none">
 We're sorry, the page you're looking for can't be found on the ADA.gov website.
 
 Please try the following:
 
 - Use the Search function to search for the words or phrases.
-- Check the <a target="blank" href="https://archive.ada.gov">Archive.ADA.gov</a> site for more ADA materials.
-</div>
+
+- Check the <a target="blank" href="https://archive.ada.gov">archive.ADA.gov</a> site for more ADA materials.
+
+
```

Comment:
supernit - extra whitespace, here and between list items.

(I think having extra whitespace between list items reads as separate lists depending on the flavor of markdown)

Reply:
fixed!